websites/F/filmmodu/dist/metadata.json

```diff
@@ -4,14 +4,20 @@
 		"name": "EGGSY",
 		"id": "162969778699501569"
 	},
+	"contributors": [
+		{
+			"name": "Fast",
+			"id": "308087335873019905"
+		}
+	],
```

Comment:
you didn't contribute to the code so you cannot be added to the contributors

Reply:
Fixed